assets/scss/_cookie.scss

```diff
@@ -9,8 +9,12 @@
     font-size: small;
     border: none;
     margin: 0;
-    
+
     a {
         color: $body-bg-colored;
     }
+
+    @media only screen and (width <= 700px) {
+        right: 0;
+    }
```

Comment:
why was this added?

Reply:
The cookie banner would leave a white space on the right side on small media queries. Now it looks complete. do I revert it?

Reply:
This seems off to me media queries should be avoided. Also i think boostrap 5 has the necessary css classes already for this

Reply:
if the banner should always be on the right, no need for media query, just me-0 should be good.

Reply:
fixed this issue. edited the right to 0 and made margin in the same file to 16px. looks better now